apps/ai-content-generator/src/components/config/configText.ts

```diff
@@ -6,10 +6,66 @@ const ModelText = {
     'capable than any previous models.',
 };
 
-const ProfileText = {
-  title: 'Describe your brand or product.',
-  textAreaPlaceholder: 'Example: Contentful is a headless content management system.',
-};
+const BrandProfileFields = [
+  {
+    id: 'profile',
+    title: 'Describe your brand or product.',
+    textAreaPlaceholder: 'Example: Contentful is a headless content management system.',
+    isRequired: true,
+    type: 'textarea',
```

Comment:
I think this a great way to organize lots of copy! nit: I am wondering if the 'type' attribute should also be 'typed' in the sense that there exist only 'textarea' | 'textinput' etc. Just to keep things strict here? Unless this is sort of a one-off detail for this particular implementation. And perhaps type could be labeled as `fieldType`?

Reply:
Great suggestions, I will update to `fieldType` and added an enum for `FieldTypes`!